projects/wvr-elements/src/lib/wvr-alert/wvr-alert.component.html

```diff
@@ -0,0 +1,64 @@
+<div
```

Comment:
lets add an `ngIf="alertClosed"` to this

Reply:
Added